db/migrations/20241011084507_AddEnvironmentCreated.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+alter table templates add column rhsm_environment_created boolean NOT NULL DEFAULT false;
+update templates set rhsm_environment_created = true;
```

Comment:
existing templates *should* have had an environment created, but in the odd case they haven't, do you think we should check that at least one update-template-content task has completed before setting this to true?

Reply:
I think this should be pretty easy to do in a query, let me see

Reply:
updated query here:

https://github.com/content-services/content-sources-backend/pull/846/files#diff-cbb9bfb9e61fbfb4cc6348f60e645c0f16ff1d42c43ce7621eb4c3dd5cc2f7deR4-R5